ci-operator/config/tnozicka/openshift-acme/tnozicka-openshift-acme-master.yaml

```diff
@@ -14,8 +14,11 @@ base_images:
     tag: machine-os-content
 
 build_root:
-  project_image:
-    dockerfile_path: images/build/Dockerfile
```

Comment:
was flaking here: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/tnozicka_openshift-acme/79/pull-ci-tnozicka-openshift-acme-master-unit/67
and I don't think we use the nss_wrapper in the end